content/pages/contribution-guide/writing-content/index.md

```diff
@@ -80,18 +80,74 @@ A featured image can be added in the following ways from the highest to lowest p
   featureAsset: "img/featured/image.webp"
   ```
 
+Please have your featured image converted to WebP as requested in [Use WebP for raster images](#use-webp-for-raster-images).
+
 [blowfish-front-matter]: https://blowfish.page/docs/front-matter/
 
 ## Use additional content types
 
-Apart from the usual content types supported by markdown, such as visuals or code blocks, you can use other content types enabled by Hugo shortcodes. This section briefly introduces them.
+Apart from the usual content types supported by markdown, such as visuals or code blocks, you can use other content types enabled by Hugo shortcodes. This section briefly introduces the most relevant shortcodes implemented on the Espressif Developer Portal.
+
+In addition to that, you can also use the standard Hugo [embedded shortcodes](https://gohugo.io/content-management/shortcodes/#embedded-shortcodes) and [Blowfish theme shortcodes](https://blowfish.page/docs/shortcodes/).
 
-If you need other types of content, either create a discussion on GitHub or offer a PR with the required functionality. It will be very much appreciated!
+If you need other content types or shortcodes implemented, either create a discussion on GitHub or offer a PR with the required functionality. It will be very much appreciated!
 
 
 ### Images
 
+#### Adding images
+
+You can add an image using a standard markdown syntax:
+
+```md
+![Alt text](path/to/image.webp "Optional tooltip")
+```
+
+However, the Blowfish theme used on this website provides the [figure](https://blowfish.page/docs/shortcodes/#figure) shortcode that offers more control over image display and render. An example of a shortcode is given below, but more parameters are available:
+
+```md
+{{</* figure
+    src="image.webp"
+    alt=""
+    caption=""
+    */>}}
+```
+
+By default, the Blowfish theme optimizes the images for different device resolutions. For some images, the processing **adds grey background**. To fix it, disable the processing of such images by adding the parameter `default="false"` to the figure shortcode.
+
+
+#### Use WebP for raster images
```

Comment:
Typo: raster -> faster
We can use "lighter" as well.

Reply:
I intentionally used the word raster here, because WebP is a raster graphics file format as opposed to vector formats, such as SVG.

Regarding _faster_, I mention it below in the line `It (i.e. WebP) allows serving web pages faster`